client/js/services/auth.js

```diff
@@ -0,0 +1,7 @@
+/* global angular */
+angular.module('mean.system')
+  .factory('AuthenticationService', ['$http',
+    $http => ({
+      login(userEmail, userPassword) { return $http.post('/api/auth/login', { userEmail, userPassword }); },
```

Comment:
I was testing this, and realized the server is expecting "user" and "password" not "userEmail" and "userPassword"